engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EventRepository.java

```diff
@@ -85,4 +85,8 @@ public List<Event> findEventsToExport(final String orgId, final LocalDate from,
 
         return findEventsRanged.getResultList();
     }
+
+    public void updateDrawerNotification(Event event) {
+        statelessSessionFactory.getCurrentSession().update(event);
```

Comment:
Not mandatory: I prefer when code performing DB operations is obvious and can be understood immediately, without having to read several methods to understand what will be modified in the DB. It can make the maintenance easier. Feel free to disagree 😄 
```suggestion
        String hql = "UPDATE Event SET renderedDrawerNotification = :renderedDrawerNotification WHERE id = :id";
        statelessSessionFactory.getCurrentSession().createQuery(hql)
                .setParameter("id", event.getId())
                .setParameter("renderedDrawerNotification", event.getRenderedDrawerNotification())
                .executeUpdate();
```

Reply:
That's because we're mixing transactional queries with non-transactional ones. Can you try:
- adding `@Transactional` to `DrawerNotificationRepository#create`
- adding `@Transactional` to `EventRepository#updateDrawerNotification`
- my suggestion above with HQL instead of update
?

Reply:
That's because we're mixing transactional queries with non-transactional ones. Let's try to improve that later. I don't want to delay the merge of this PR because of it.